src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -214,45 +214,83 @@ This resets the cluster to its initial state.`,
 				return err
 			}
 
-			var eg errgroup.Group
+			forward := func(userOutput chan string) error {
```

Comment:
Is there a reason we need to use a channel here? It seems like we're just using it to get all the log lines to a goro so they can be printed but we could just print them at the point we send them through the channel and I think we'd get semantically equivalent code.

Reply:
I tried that @jdoliner, but I don't think that the error Group allows this. Or else, I was doing something else wrong. Things just ran in the background, because it's only in the case that all in the group returned that something happened. I could definitely be wrong here, because I'm new to error groups. But I did try that without success. Any pointers?